### PR TITLE
Houston hotfix RPIP draft

### PR DESCRIPTION
**Pull request by darrenlangley**

### CODE REVIEW

File: RPIPs/RPIP-HoustonHotfix.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- Two high bug bounties that enabled someone to steal a proposer's bond by preventing them from responding to challenges 
+While working on the bug bounties we decided to refactor how the voting system burns a veto'd proposer bond. We decided to use RPL's burn function rather than transferring to the 0x0 address so that token supply is recorded accurately. 
+
+Additionally, the community discovered that the Protocol DAO quorum guardrail is set too high, as it should be possible to match it with RPIP-4. The community also pointed out some small parameter name inconsistencies. 
+
+There has also been discussion around a potential tokenomics rework prelude [RPIP-62](https://rpips.rocketpool.net/RPIPs/RPIP-62) that uses a combination of setting changes and reward tree changes, to deliver some of the value of Saturn earlier. One issue with rolling out these changes is that they would break the RPL scrub penalty that prevents a potential security exploit. Ultimately the scrub penalty will be removed in Saturn due to an improved proof-based mechanism but in the meantime it is needed. To give us the option to support RPIP-62, the scrub penalty change as been included in this hotfix.
```
Comment (Valdorff) [inline RPIPs/RPIP-HoustonHotfix.md:36]: > the scrub penalty change as been included in this hotfix.
typo "has"

File: RPIPs/RPIP-HoustonHotfix.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+### Finalising Legacy Minipools Issue
+
+- `RocketMinipoolManager` SHALL calculate ETH matched correctly and update the network snapshots.
+- A function SHALL be provided on the upgrade contract to fix any node operators who have experienced the ETH matched bug
+
+### Parameter Fixes
+
+- The `RocketDAOProtocolSettingsAuction` `Lot Duration` guardrail SHALL use blocks instead of seconds 
+- When setting the `RocketDAOProtocolSettingsNode` `MinimumPerMinipoolStake` parameter to 0, `RocketNodeStaking` `getNodeETHMatchedLimit` SHALL NOT revert due to division by 0
+- Some other minor fixes to parameter names to make them consistent and remove typos
```
Comment (Valdorff) [inline RPIPs/RPIP-HoustonHotfix.md:49]: - Non-functional fixes to improve parameter names to and remove typos MAY be included

**Review by Valdorff — APPROVED** [review]
LGTM for Draft to get forum feedback. Approving.
A couple of minor comments, but not critical when they get in.

File: RPIPs/RPIP-HoustonHotfix.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Abstract
+
+The Houston hotfix does not introduce new features but does include important changes to existing features that:
+
+- Fix a discovered bug with finalising legacy minipools
+- Resolves Immunefi bug bounty submissions regarding the onchain voting system
+- Improves Protocol DAO quorum default guardrail value (currently set too high)
+- Fixes a couple of issues with Protocol DAO parameters
+- Refactored how the onchain voting system burns a veto'd proposer bond - now actually burns the RPL rather than transferring it to 0x0 address
```
Comment (kanewallmann) [inline RPIPs/RPIP-HoustonHotfix.md:21]: Change "Refactored" to "Modified" as refactored implies there is no change to external behaviour

File: RPIPs/RPIP-HoustonHotfix.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+- Miscellaneous housekeeping of parameter names to ensure consistency 
+
+## Motivation
+
+Soon after the Houston launch, an issue was reported with finalising legacy minipools. The issue was introduced by the new onchain voting snapshoting system that ensures proposal quorum information can be accessed for historical blocks onchain. A smart node release was deployed to contain the issue but it was decided that a smart contract hotfix would be released to resolve the issue.
+
+In the meantime, 3x Immunefi bug bounties were raised regarding the onchain voting system.
+- A medium bug bounty that enabled a node operator to exit RPL stake via the onchain proposal challenge system.
+- Two high bug bounties that enabled someone to steal a proposer's bond by preventing them from responding to challenges 
+While working on the bug bounties we decided to refactor how the voting system burns a veto'd proposer bond. We decided to use RPL's burn function rather than transferring to the 0x0 address so that token supply is recorded accurately.
```
Comment (kanewallmann) [inline RPIPs/RPIP-HoustonHotfix.md:32]: Missing a period here and change "refactor" to "modified"

File: RPIPs/RPIP-HoustonHotfix.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- When setting the `RocketDAOProtocolSettingsNode` `MinimumPerMinipoolStake` parameter to 0, `RocketNodeStaking` `getNodeETHMatchedLimit` SHALL NOT revert due to division by 0
+- Some other minor fixes to parameter names to make them consistent and remove typos
+
+### Protocol Quorum Guardrail
+
+- The `RocketDAOProtocolSettingsProposals` `Proposal Quorum` minimum safeguard SHALL be changed from 51% to 15%
+
+### Immunefi Bug Bounties
+
+- On a successful challenge 20% of the proposers bond SHALL be burnt and the remaining distributed to challengers
```
Comment (kanewallmann) [inline RPIPs/RPIP-HoustonHotfix.md:57]: proposer's*

**Review by kanewallmann — COMMENTED** [review]
Good stuff

File: RPIPs/RPIP-HoustonHotfix.md (hunk 5 above)
Comment (darrenlangley) [reply RPIPs/RPIP-HoustonHotfix.md:57]: damn I am usually good at picking those up, thanks!

**Pull request merged**